contracts/ens/ArgentENSManager.sol

```diff
@@ -127,10 +119,18 @@ contract ArgentENSManager is IENSManager, Owned, Managed, ENSConsumer {
      */
     function isAvailable(bytes32 _subnode) public view returns (bool) {
         bytes32 node = keccak256(abi.encodePacked(rootNode, _subnode));
-        address currentOwner = getENSRegistry().owner(node);
+        address currentOwner = ensRegistry.owner(node);
         if(currentOwner == address(0)) {
             return true;
         }
         return false;
     }
+
+    /**
+    * @dev Gets the official ENS reverse registrar. 
+    * @return Address of the ENS reverse registrar.
+    */
+    function getENSReverseRegistrar() external view returns (address) {
```

Comment:
Now that we no longer use `ENSConsumer` this method should be internal or removed all-together (I prefer that option). It would save some gas and make the code cleaner.

L106 could be
```
ENSReverseRegistrar reverseRegistrar = ENSReverseRegistrar(ensRegistry.owner(ADDR_REVERSE_NODE));
```